app/views/dashboard/ng-partials/widgets/_move.html.haml

```diff
@@ -14,7 +14,7 @@
             %button.move-amount.no-radius.rightmost{stepsize: '1000'} 1000
 
         .row
-          %table.transparent-table.jog-table{:align => "center", :style => 'border: 0px;'}
+          %table.jog-table{:align => "center", :style => 'border: 0px;'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace